libraries/core_libs/storage/src/storage.cpp

```diff
@@ -59,8 +64,10 @@ DbStorage::DbStorage(fs::path const& path, uint32_t db_snapshot_each_n_pbft_bloc
   if (db_revert_to_period) {
     recoverToPeriod(db_revert_to_period);
   }
-
-  checkStatus(rocksdb::DB::Open(options, db_path_.string(), descriptors, &handles_, &db_));
+  rocksdb::DB* db = nullptr;
+  checkStatus(rocksdb::DB::Open(options, db_path_.string(), descriptors, &handles_, &db));
+  assert(db);
+  db_.reset(db);
```

Comment:
why not use "rocksdb::DB* db_" directly? What is the benefits using "std::unique_ptr<rocksdb::DB> db_"? Thanks

Reply:
so we do not need to take care of deleting it, in case some code change